app/scripts/modules/core/naming/naming.service.ts

```diff
@@ -0,0 +1,90 @@
+import {module} from 'angular';
+
+export const NAMING_SERVICE = 'spinnaker.core.naming.service';
```

Comment:
where do we want to put these?  IMO, it makes sense to put it next to the module definition unless the name is used in the class(es) contained in the file.  thoughts?